Lib/email/utils.py

```diff
@@ -181,6 +177,11 @@ def make_msgid(idstring=None, domain=None):
     portion of the message id after the '@'.  It defaults to the locally
     defined hostname.
     """
+    # Lazy imports to speedup module import time
+    # (no other functions in email.utils need these modules)
+    import os
+    import random
+    import socket
```

Comment:
Style:

```suggestion
    import socket

```

(I think ``Charset`` can get away without the newline as there's only one line in the pertinent block; here we have a whole function.